### DISCUSSION TITLE
Is there a way to prevent fallbackData fetching on client side initially?

### DISCUSSION THREAD

**Discussion by Absanater** [vercel/swr#1550] [Q&A] [answered]
I have been using looking to upgrade my swr version however noticed after changing initialData to fallbackData the hook would recall the endpoint client side. Is there a way to achieve what initalData originally did to prevent the duplicate call?

**Comment by shuding** [comment, marked as answer]
I think `revalidateIfStale: false` is what you need. The downside is the data will not be refreshed on hydration (if that’s ok for you).

**Reply by hazae41** [reply]
Well he can revalidate on hydratation/mount by using an effect

```typescript
useEffect(() => {
  mutate()
}, [])
```

If you do so, be sure to disable `revalidateOnMount`
